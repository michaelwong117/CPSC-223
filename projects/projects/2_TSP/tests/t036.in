256
RFD NLA AUM KLL GNA UST BHI MS1 05A UOX DUR AMP OAJ BYU KKI BXE PCG WBR CXY AFA IZO PWT LBQ SUU TKK LTO AMH MBK LAS OOM TDZ MCI CBF IFL CIH ENV FDY WA7 HAB KGS SQV MLD TGM TIX SMN HZL FOC CGX MBZ VLL HGZ PHN PMI PCD INT LKV PLZ YUM SCM ERD HZH NCL LRL KIV RRG TJM GDL CA4 PAQ ULN TIE ILZ KLU BSG YOP JMN MUT VGC ME3 CMG MMH BID BMA YXZ KUA UKB SCQ LUG CXH VWT GGN NBC TUC DWA SXL RAC MOG TCQ GOH FRD CAH SIK MLT BMJ VST PTK GRP GFA GRA KSC CCC EVM BHX VWW ZXE YOD APC IDI NKI AXV THV ARE SXM BXU AZ3 PHI FRF BOY K01 SZF DIG KHV GUF OCE CPT OHD MGC AVX IAD PQQ FQJ MS3 HDF OSU PGM RDA JEF 1G4 RBF BKK BIR LAR FKB BYK HSV PUB CFR USH ZPB ELA CGC PGH A56 KBS YZH CAP PCE CZE XRH DTR CPO A69 A61 GAU PET WFK PTR ORI LHW PDS DEL MWL BGI BCB CHT OSP URG KWJ TVC ILL SKW ITM TAL MMQ ZCO DIV MTD DXR A03 BBS MZG MSH DUB CNE LOI HND SLS LUE BRY TOD CEG MJV UDD GLN A87 CIA VQS NKX NAP YSP THA OMO DUD WIC BJU 04A PNR CCY RAE MWD ARM MSP FYV SNR BBF UXM BVB LNP SBW ZBX HPT CJC AK3 SAV HUH LAK TFF WSZ A94 MIC RSA YGX JFN A53 A07 LAO
42.195278 -89.097222
-12.998333 28.665000
43.662778 -92.933056
59.128056 -156.858611
53.601944 24.053889
29.959167 -81.339722
-38.725556 -62.168889
33.832500 -89.798056
67.570000 -148.183889
34.384444 -89.536944
-29.611667 31.119444
-24.699444 44.732778
34.829167 -77.612222
49.984722 11.640000
60.913889 -161.493333
14.847222 -12.468333
17.261111 -90.246111
43.722778 -85.504167
25.554444 -79.275278
-34.587778 -68.403333
35.413611 132.890000
47.490278 -122.764722
-0.705556 10.244167
38.264444 -121.924167
7.461944 151.843056
25.990000 -111.348333
6.039444 37.590833
-10.233333 -54.950000
36.080000 -115.152222
-36.308333 148.973333
41.650000 -83.533333
39.297500 -94.713889
41.260000 -95.758611
-17.559444 146.011667
36.183333 113.100000
40.718611 -114.030833
41.011944 -83.668611
47.833333 -120.033333
34.129167 -87.994444
36.794722 27.091111
48.098056 -123.187222
42.170556 -112.289444
46.467778 24.412500
28.514722 -80.799167
45.123889 -113.881389
40.986667 -75.995000
25.934722 119.663611
41.858889 -87.608333
-3.372222 -57.724722
41.706389 -4.851944
66.191667 -155.668889
42.910833 -82.528889
39.551667 2.738889
43.019167 -91.123611
36.133611 -80.221944
42.161111 -120.399167
-33.985000 25.617222
32.656667 -114.605833
61.844444 -165.573611
46.733333 36.783333
26.322222 109.150000
55.038056 -1.689722
9.767222 1.091111
46.927778 28.930833
-19.757778 63.361111
57.179722 65.319444
20.521944 -103.311111
41.777500 -82.678056
61.595000 -149.088611
47.842500 106.767500
7.201111 35.415278
49.233333 18.613611
46.642500 14.337778
1.902500 9.805833
58.491389 -119.408333
44.163889 -94.004167
41.367778 -91.148333
42.843333 -75.561111
44.409444 -69.011944
-19.011944 -57.673056
37.631111 -118.851389
41.168056 -71.577778
59.354444 17.941389
47.966944 -84.786389
3.775278 103.209167
34.632778 135.223889
42.896389 -8.415278
46.004167 8.910556
49.300000 -123.133333
32.110833 -84.188889
6.103611 -5.986944
55.564722 52.092500
-26.833333 -65.200000
38.579444 -121.856944
54.280278 -8.599167
42.761111 -87.813889
20.518056 99.258333
-18.053333 -70.275833
64.190833 -51.678056
48.521944 -123.024444
9.177500 105.177778
36.898889 -89.561667
45.647778 -68.685556
7.370833 -60.489722
59.589444 16.633611
42.665556 -83.420556
-11.739722 -49.132222
47.504722 -111.187222
8.328333 -73.691944
48.663056 21.241111
22.460833 -78.328611
47.425000 -92.498056
52.453889 -1.748056
46.386667 -112.764444
52.845833 0.766111
54.405000 -110.279444
38.213056 -122.280556
40.632222 -79.105556
55.875000 -133.220833
40.493611 -84.298056
39.916944 -76.873056
18.450000 -66.675278
18.040833 -63.108889
8.951389 125.483889
33.526944 -112.295278
-2.474722 -45.105278
50.030278 8.588056
11.159722 -4.331111
36.800000 -86.810278
41.254444 36.567222
27.792222 99.678056
48.528056 135.188333
30.289722 -87.671667
38.310556 -75.123889
-33.964722 18.601667
41.180000 20.742222
41.703333 -86.821111
33.405000 -118.415833
38.947500 -77.460000
-31.435833 152.863333
58.098611 -134.760000
32.799167 -89.126111
53.878889 14.151389
40.079722 -83.073056
59.348333 -151.831667
-18.950000 135.200000
38.591111 -92.156111
35.986111 -113.816944
34.263889 -116.856111
13.680833 100.747778
26.482222 87.263611
41.311944 -105.675000
48.779444 8.080556
7.738889 -5.073889
34.637222 -86.775000
38.290000 -104.498056
49.173333 -0.450000
-54.802778 -68.327778
53.891111 -92.196389
29.600556 -96.321944
-5.452778 148.423611
29.033611 79.472222
56.036667 -160.266111
7.944444 -11.761111
55.293056 -114.777222
19.732778 -72.194722
57.153889 -157.396944
11.414722 -69.681667
-33.600556 150.780833
48.510000 -122.803056
-27.261111 -70.779167
55.740000 -162.125278
54.582500 -164.914444
26.107778 91.585833
-31.718333 -52.327778
47.285556 -68.312778
57.833333 -153.250000
57.885278 -152.846111
36.515278 103.620833
28.627500 -100.535278
28.559444 77.099444
32.781667 -98.060278
13.074722 -59.492222
37.207778 -80.407778
-43.811944 -176.465556
54.478611 17.104167
-29.782222 -57.038056
35.126389 126.808889
44.741667 -85.582222
45.116944 -95.129444
61.965278 -151.191389
34.785556 135.438056
65.174444 -152.108056
-8.858056 31.334444
-38.766944 -72.637222
5.833333 -5.366667
-16.978333 130.555000
41.371667 -73.482222
58.457500 -154.023333
51.323889 -0.847500
23.569167 119.618611
20.673333 58.891944
53.421389 -6.270000
38.428333 -105.106111
27.615556 -99.520000
35.553333 139.781111
44.055000 27.180000
48.340000 19.735556
37.814444 -85.499722
2.818056 104.160000
53.178056 -2.977778
37.778889 -0.813333
33.748333 -116.274722
29.026389 -10.050556
58.583056 -152.445833
41.799444 12.595000
18.134722 -65.493611
32.868333 -117.141667
40.884444 14.290833
48.757222 -86.345833
35.380000 -86.246667
43.282778 17.845833
-45.929167 170.197500
58.458889 -3.093056
29.503611 81.668333
68.083333 -163.166667
-4.816111 11.886667
43.072500 -92.610833
30.906667 41.138333
32.563056 71.570556
-30.528056 151.617222
44.881944 -93.221667
36.005000 -94.170000
47.312222 -2.149167
42.508333 -71.183333
61.044444 -158.145278
2.846389 -60.690000
36.987500 -82.530000
2.261944 111.985556
36.651667 -93.345833
42.723611 -93.226389
-22.498333 -68.903611
64.198056 -147.928611
32.127500 -81.202222
-16.686944 -151.021667
68.222222 -134.985833
-3.383056 -64.724167
-41.737222 171.579167
70.574167 -155.318056
45.061944 -93.353889
-36.588056 -64.275556
56.357778 -94.710833
41.778056 -80.695556
64.333333 -147.500000
66.766111 -160.152778
18.176111 120.531111
