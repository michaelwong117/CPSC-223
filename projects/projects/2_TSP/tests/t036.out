-exchange adjacent:   1570625.91 RFD NLA LAO A07 A53 YGX JFN RSA MIC A94 HUH WSZ TFF AK3 LAK SAV CJC SBW HPT ZBX LNP BVB SNR UXM MSP BBF FYV ARM MWD RAE PNR CCY 04A BJU OMO WIC DUD NKX YSP THA VQS CIA NAP A87 UDD GLN MJV CEG BRY TOD SLS LUE HND CNE LOI DUB MZG MSH BBS DXR A03 MTD ZCO DIV MMQ ITM TAL SKW ILL TVC OSP KWJ CHT URG BGI BCB PDS MWL DEL LHW PTR ORI WFK PET GAU A61 A69 DTR CPO XRH PCE CAP CZE KBS YZH A56 CGC PGH ZPB ELA USH CFR PUB HSV BYK FKB LAR BIR BKK RBF 1G4 JEF PGM RDA HDF OSU MS3 FQJ PQQ AVX MGC IAD OHD CPT OCE GUF KHV DIG SZF FRF BOY PHI K01 BXU AZ3 ARE SXM THV IDI AXV VWW APC NKI YOD BHX ZXE KSC EVM CCC GRA GRP PTK GFA VST MLT BMJ TCQ SIK FRD CAH GOH SXL MOG DWA RAC TUC GGN NBC CXH VWT SCQ LUG KUA UKB BMA BID YXZ MMH CMG ME3 VGC MUT JMN YOP BSG KLU ILZ TIE ULN PAQ GDL CA4 TJM RRG LRL NCL KIV ERD HZH SCM LKV YUM INT PLZ PHN PCD HGZ PMI VLL MBZ CGX FOC SMN TIX HZL TGM SQV MLD KGS FDY HAB ENV WA7 CIH IFL LAS CBF MCI TDZ MBK AMH OOM TKK LTO SUU PWT IZO LBQ AFA PCG CXY WBR BXE BYU KKI OAJ DUR AMP 05A UOX MS1 UST BHI GNA KLL AUM RFD
